src/nccl_ofi_rdma.c

```diff
@@ -501,7 +505,13 @@ static inline int get_properties(nccl_net_ofi_device_t *base_dev,
 		static_assert(NCCL_OFI_RDMA_COMM_ID_BITS < 31,
 					  "NCCL_OFI_RDMA_COMM_ID_BITS must be less than 31 so max_communicators fits in an integer");
 		props->max_communicators = NCCL_OFI_RDMA_MAX_COMMS;
+	} else {
+		return ret;
 	}
+
+	props->rma_supported = 1;
+	props->max_write_inline_size = max_write_inline_size;
```

Comment:
Nit: could assert `is_max_write_inline_size_initialized` here.

Reply:
Applied and updated...